src/django/api/migrations/0107_alter_trainedmodel_is_active.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.4 on 2022-08-24 18:26
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0106_trainedmodel_unique_is_active'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='trainedmodel',
+            name='is_active',
+            field=models.BooleanField(default=False, help_text='True if this is the currently active version of the dedupe         model'),
+        ),
+    ]
```

Comment:
Changing the way the help_text is broken across 2 lines should allow us to delete this migration (see comment in the models file)